stanford/bin/cut-release-branch

```diff
@@ -13,7 +13,9 @@ done
 : ${BRANCH_TO_CUT:=rc}
 : ${WORKSPACE:=${HOME}/workspace/cut_branch}
 : ${EMAIL_RECIPIENT:=''}
-date=$(echo "${BUILD_ID}" | head -c10)
+: ${EMAIL_SENDER:= 'jenkins@build.lagunita.stanford.edu'}
+: ${EMAIL_SENDER_NAME:= 'Jenkins'}
```

Comment:
I'd prefer this as R C Cutter as it's more about what it is, is more distinguishable and it stands out more in my mail. Having said that, it's not something I'm overly attached to.

Other stuff is fine.